mantidproject/mantid
# ISISPowderDiffractionPolarisTest: Add test case with ExistingV=yes

fixes #14560

The system test previously was running with only `ExistingV=no`. Now that Ron has provided us with the correct calibration files, I have integrated those .cal files with a separate test case which runs the Polaris system test with ExistingV=yes. The nexus files which are going to be reused enables the script to go through a shorter workflow as well.

**To Test**
- Run the test locally
- Make sure all files mentioned in system tests are cleared on completion

(_NOTE: modified `ISISPowderDiffractionHrpdTest.py` so that `ISISPowderDiffractionHrpd2` uses the correct test cases, which is `LoadTests2`_)
